### ISSUE TITLE
Heroku assets:precompile error

### ISSUE THREAD

**Issue by cseydel** [rubysherpas/forem#530]
Hi,
I am trying to deploy my app to heroku but it fails for forem base.css.

```
Preparing app for Rails asset pipeline
       Running: rake assets:precompile
       [deprecated] I18n.enforce_available_locales will default to true in the future. If you really want to skip validation of your locale you can set I18n.enforce_available_locales = false to avoid this message.
       I, [2014-02-10T01:25:39.676601 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/forem/lock_icon-a5d93f3ed5bbd863f2d95e070196648b.png
       I, [2014-02-10T01:25:39.678473 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/forem/pin_icon-21be9c88e6e1f29ff03565462382cad1.png
       I, [2014-02-10T01:25:49.980246 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/admin-128f67739ff01fb4a2ce435c97abc699.js
       I, [2014-02-10T01:26:10.398974 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/application-934ee99ce6a359ff4bc2afc9a7de30d0.js
       I, [2014-02-10T01:26:17.213279 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/admin-1844e8c00924061f43606b5256d27f79.css
       I, [2014-02-10T01:26:19.786749 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/admin_base-2391f8d7dcf7b9faba0aebce27e0f4b3.css
       I, [2014-02-10T01:26:20.603998 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/application-7bdf7e15e94bec0e88e5f5b3a1c0bccf.css
       I, [2014-02-10T01:26:20.760950 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/base-6aa4789d5ba9d382e1d956bef843d320.css
       I, [2014-02-10T01:26:22.275572 #1037]  INFO -- : Writing /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/public/assets/forem-b1b8acbe496fd1dcb5106f59e1fd7f7e.js
       rake aborted!
       undefined method `image_path' for nil:NilClass
       (in /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/bundler/gems/forem-c48dd3bba333/app/assets/stylesheets/forem/base.css)
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/sass_functions.rb:18:in `image_url'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/script/funcall.rb:113:in `_perform'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/script/node.rb:40:in `perform'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/script/list.rb:71:in `block in _perform'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/script/list.rb:71:in `map'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/script/list.rb:71:in `_perform'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/script/node.rb:40:in `perform'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:298:in `visit_prop'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:37:in `visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:100:in `visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:53:in `block in visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:53:in `map'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:53:in `visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:109:in `block in visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:121:in `with_environment'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:108:in `visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:37:in `block in visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:320:in `visit_rule'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:37:in `visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:100:in `visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:53:in `block in visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:53:in `map'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:53:in `visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:109:in `block in visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:121:in `with_environment'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:108:in `visit_children'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:37:in `block in visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:128:in `visit_root'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/base.rb:37:in `visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:100:in `visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/visitors/perform.rb:7:in `visit'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/tree/root_node.rb:20:in `render'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/engine.rb:315:in `_render'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sass-3.2.14/lib/sass/engine.rb:262:in `render'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/sass_compressor.rb:24:in `evaluate'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/tilt-1.4.1/lib/tilt/template.rb:103:in `render'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/context.rb:197:in `block in evaluate'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/context.rb:194:in `each'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/context.rb:194:in `evaluate'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/bundled_asset.rb:25:in `initialize'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/base.rb:377:in `new'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/base.rb:377:in `build_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/index.rb:94:in `block in build_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/caching.rb:58:in `cache_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/index.rb:93:in `build_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/base.rb:287:in `find_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/index.rb:61:in `find_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/manifest.rb:211:in `block in find_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/manifest.rb:257:in `benchmark'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/manifest.rb:210:in `find_asset'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/manifest.rb:119:in `block in compile'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/manifest.rb:118:in `each'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/sprockets/manifest.rb:118:in `compile'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-rails-2.0.1/lib/sprockets/rails/task.rb:60:in `block (3 levels) in define'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-2.10.1/lib/rake/sprocketstask.rb:146:in `with_logger'
       /tmp/build_6ee0543f-6ad1-48d9-84d1-1d7502881e1f/vendor/bundle/ruby/2.0.0/gems/sprockets-rails-2.0.1/lib/sprockets/rails/task.rb:59:in `block (2 levels) in define'
       Tasks: TOP => assets:precompile
       (See full trace by running task with --trace)
 !
 !     Precompiling assets failed.
 !

 !     Push rejected, failed to compile Ruby app

```

with this gems:

```
GIT
  remote: git://github.com/radar/forem.git
  revision: c48dd3bba33329f04ac93b663bfe74a997cc4ed9
  branch: rails4
  specs:
    forem (1.0.0.beta1)
      decorators (~> 1.0.2)
      friendly_id (= 5.0.2)
      gemoji (= 1.1.2)
      rails (~> 4.0.0)
      sanitize (= 2.0.6)
      select2-rails (~> 3.4.3)
      simple_form (~> 3.0.0.rc)
      workflow (= 1.0.0)

GIT
  remote: https://github.com/nukturnal/cancan.git
  revision: a606378912e75d6b62f8257ffa5b4c5f2023760b
  specs:
    cancan (1.6.10)

GEM
  remote: https://rubygems.org/
  specs:
    aasm (3.0.26)
    actionmailer (4.0.2)
      actionpack (= 4.0.2)
      mail (~> 2.5.4)
    actionpack (4.0.2)
      activesupport (= 4.0.2)
      builder (~> 3.1.0)
      erubis (~> 2.7.0)
      rack (~> 1.5.2)
      rack-test (~> 0.6.2)
    activemodel (4.0.2)
      activesupport (= 4.0.2)
      builder (~> 3.1.0)
    activerecord (4.0.2)
      activemodel (= 4.0.2)
      activerecord-deprecated_finders (~> 1.0.2)
      activesupport (= 4.0.2)
      arel (~> 4.0.0)
    activerecord-deprecated_finders (1.0.3)
    activesupport (4.0.2)
      i18n (~> 0.6, >= 0.6.4)
      minitest (~> 4.2)
      multi_json (~> 1.3)
      thread_safe (~> 0.1)
      tzinfo (~> 0.3.37)
    arel (4.0.2)
    atomic (1.1.14)
    auto_html (1.6.1)
      redcarpet (~> 2.0)
      rinku (~> 1.5.0)
    aws-s3 (0.6.3)
      builder
      mime-types
      xml-simple
    aws-sdk (1.5.8)
      httparty (~> 0.7)
      json (~> 1.4)
      nokogiri (>= 1.4.4)
      uuidtools (~> 2.1)
    bcrypt-ruby (3.1.2)
    bootstrap-sass (3.1.0.2)
      sass (~> 3.2)
    builder (3.1.4)
    chunky_png (1.2.9)
    climate_control (0.0.3)
      activesupport (>= 3.0)
    cocaine (0.5.3)
      climate_control (>= 0.0.3, < 1.0)
    coffee-rails (4.0.1)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0, < 5.0)
    coffee-script (2.2.0)
      coffee-script-source
      execjs
    coffee-script-source (1.7.0)
    compass (0.12.2)
      chunky_png (~> 1.2)
      fssm (>= 0.2.7)
      sass (~> 3.1)
    compass-rails (1.1.3)
      compass (>= 0.12.2)
    dalli (2.6.4)
    decorators (1.0.3)
      railties (>= 3.0.19, != 3.2.9, != 3.2.8, != 3.2.7, != 3.2.6, != 3.2.5, != 3.2.4, != 3.2.3, != 3.2.2, != 3.2.10, != 3.2.1, != 3.2.0, != 3.1.9, != 3.1.8, != 3.1.7, != 3.1.6, != 3.1.5, != 3.1.4, != 3.1.3, != 3.1.2, != 3.1.1, != 3.1.0)
    devise (3.2.2)
      bcrypt-ruby (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 3.2.6, < 5)
      thread_safe (~> 0.1)
      warden (~> 1.2.3)
    erubis (2.7.0)
    execjs (2.0.2)
    friendly_id (5.0.2)
      activerecord (~> 4.0.0)
    fssm (0.2.10)
    gemoji (1.1.2)
    gravtastic (2.2.0)
    haml (4.0.5)
      tilt
    has_scope (0.6.0.rc)
      actionpack (>= 3.2, < 5)
      activesupport (>= 3.2, < 5)
    hike (1.2.3)
    httparty (0.12.0)
      json (~> 1.8)
      multi_xml (>= 0.5.2)
    i18n (0.6.9)
    inherited_resources (1.4.1)
      has_scope (~> 0.6.0.rc)
      responders (~> 1.0.0.rc)
    jquery-rails (3.0.2)
      railties (>= 3.0, < 5.0)
      thor (>= 0.14, < 2.0)
    jquery-ui-rails (4.1.1)
      railties (>= 3.1.0)
    json (1.8.1)
    kaminari (0.14.1)
      actionpack (>= 3.0.0)
      activesupport (>= 3.0.0)
    kgio (2.9.1)
    mail (2.5.4)
      mime-types (~> 1.16)
      treetop (~> 1.4.8)
    mail_view (2.0.4)
      tilt
    memcachier (0.0.2)
    mime-types (1.25.1)
    mini_portile (0.5.2)
    minitest (4.7.5)
    multi_json (1.8.4)
    multi_xml (0.5.5)
    nokogiri (1.6.1)
      mini_portile (~> 0.5.0)
    orm_adapter (0.5.0)
    paperclip (3.5.4)
      activemodel (>= 3.0.0)
      activesupport (>= 3.0.0)
      cocaine (~> 0.5.3)
      mime-types
    pg (0.17.1)
    polyglot (0.3.3)
    protected_attributes (1.0.5)
      activemodel (>= 4.0.1, < 5.0)
    rack (1.5.2)
    rack-cache (1.2)
      rack (>= 0.4)
    rack-test (0.6.2)
      rack (>= 1.0)
    rails (4.0.2)
      actionmailer (= 4.0.2)
      actionpack (= 4.0.2)
      activerecord (= 4.0.2)
      activesupport (= 4.0.2)
      bundler (>= 1.3.0, < 2.0)
      railties (= 4.0.2)
      sprockets-rails (~> 2.0.0)
    rails_12factor (0.0.2)
      rails_serve_static_assets
      rails_stdout_logging
    rails_autolink (1.1.0)
      rails (> 3.1)
    rails_serve_static_assets (0.0.2)
    rails_stdout_logging (0.0.3)
    railties (4.0.2)
      actionpack (= 4.0.2)
      activesupport (= 4.0.2)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    raindrops (0.12.0)
    rake (10.1.1)
    redcarpet (2.3.0)
    responders (1.0.0)
      railties (>= 3.2, < 5)
    rinku (1.5.1)
    sanitize (2.0.6)
      nokogiri (>= 1.4.4)
    sass (3.2.14)
    sass-rails (4.0.1)
      railties (>= 4.0.0, < 5.0)
      sass (>= 3.1.10)
      sprockets-rails (~> 2.0.0)
    select2-rails (3.4.9)
      sass-rails
      thor (~> 0.14)
    simple_form (3.0.1)
      actionpack (>= 4.0.0, < 4.1)
      activemodel (>= 4.0.0, < 4.1)
    sprockets (2.10.1)
      hike (~> 1.2)
      multi_json (~> 1.0)
      rack (~> 1.0)
      tilt (~> 1.1, != 1.3.0)
    sprockets-rails (2.0.1)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      sprockets (~> 2.8)
    thor (0.18.1)
    thread_safe (0.1.3)
      atomic
    tilt (1.4.1)
    treetop (1.4.15)
      polyglot
      polyglot (>= 0.3.1)
    tzinfo (0.3.38)
    uglifier (2.3.3)
      execjs (>= 0.3.0)
      json (>= 1.8.0)
    unicorn (4.7.0)
      kgio (~> 2.6)
      rack
      raindrops (~> 0.7)
    uuidtools (2.1.4)
    warden (1.2.3)
      rack (>= 1.0)
    workflow (1.0.0)
    xml-simple (1.1.3)

PLATFORMS
  ruby

DEPENDENCIES
  aasm (~> 3.0.16)
  auto_html (~> 1.6.0)
  aws-s3
  aws-sdk (~> 1.5.7)
  bootstrap-sass
  cancan!
  coffee-rails (~> 4.0.1)
  compass-rails
  dalli (~> 2.6.4)
  devise
  forem!
  friendly_id
  gravtastic (= 2.2.0)
  haml
  inherited_resources (= 1.4.1)
  jquery-rails (= 3.0.2)
  jquery-ui-rails
  kaminari (= 0.14.1)
  mail_view (~> 2.0.1)
  memcachier
  paperclip (~> 3.5.1)
  pg (~> 0.17.0)
  protected_attributes
  rack-cache
  rails (= 4.0.2)
  rails_12factor
  rails_autolink (= 1.1.0)
  sass-rails
  simple_form
  uglifier (~> 2.3.1)
  unicorn (~> 4.7.0)
```

May someone can give me a hand on this?
To get straight, this are my application.rb lines referring to assets:

```
config.assets.enabled = true
    config.assets.version = '1.0'
    config.assets.precompile += [ Proc.new {|path| File.basename(path) =~ /^[^_].*\.\w+$/} ]
    config.assets.precompile += ['admin.js', 'bjqs-1.3.js', 'bjqs.css', 'admin.css', '*.png', '*.jpg', '*.jpeg', '*.gif']
    config.assets.initialize_on_precompile = false
```

**Comment by radar** [comment]
Hi @cseydel, sorry to hear you're having problems.

Do you have an application that I can use to try and reproduce this problem?

**Comment by emacca** [comment]
We've got the same problem... https://github.com/lukemesiti/autismconnect

We can't currently deploy to heroku because of this.

I've grepped across the project and can find no instance of image_path anywhere in the code.
It's not in our codebase. It doesn't appear to be in forem's codebase either so it's pretty confusing.

**Comment by bodrovis** [comment]
I got the same error when deploying. This is happening because https://github.com/radar/forem/blob/rails4/app/assets/stylesheets/forem/base.css here `select2` is being required (which in turn is used when adding user to a group inside Admin section. I ended up hooking up `select2` directly, without `base.css` and `select2-rails` gem. 

But this error seems to be fixed in a new version of `select2-rails` so I will submit a PR to bump it.
